#! /bin/sh /usr/share/dpatch/dpatch-run
## hide_manual_menu.dpatch by Roland Stigge <stigge@antcom.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Hide user manual menu since no manual is available.

@DPATCH@
diff -urNad linphone-1.7.1~/gtk/interface.c linphone-1.7.1/gtk/interface.c
--- linphone-1.7.1~/gtk/interface.c	2007-04-16 13:53:49.000000000 +0200
+++ linphone-1.7.1/gtk/interface.c	2007-08-13 13:41:29.000000000 +0200
@@ -205,7 +205,9 @@
   gtk_container_add (GTK_CONTAINER (help1_menu), about1);
 
   user_manual1 = gtk_image_menu_item_new_from_stock ("gtk-help", accel_group);
+#if 0
   gtk_widget_show (user_manual1);
+#endif
   gtk_container_add (GTK_CONTAINER (help1_menu), user_manual1);
 
   frame6 = gtk_frame_new (NULL);
